cmake/Modules/FindOpenBLAS.cmake

```diff
@@ -89,3 +90,21 @@ MARK_AS_ADVANCED(
     OpenBLAS
 )
 
+#   Check ILP64 data model for the case of Julia self-shipped `libopenblas64_.so`
```

Comment:
Could anyone review this cmake script?

Reply:
I don't know much, but maybe you can simplify a bit using CHECK_SOURCE_COMPILES and similar. https://gitlab.kitware.com/cmake/community/wikis/doc/tutorials/How-To-Write-Platform-Checks
Overall looks ok-ish to me.

Reply:
ah, looks greate, I wiil try 
- [ ] `CHECK_SYMBOL_EXISTS`

Reply:
Any updates?

Reply:
sorry, I'm swamped... I will try it after 1/10.